coderd/notifications/testdata/rendered-templates/smtp/TemplateTemplateDeleted.html.golden

```diff
@@ -37,8 +37,12 @@ l', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif; color: #020617=
 r: 1px solid #e2e8f0; border-radius: 8px; background-color: #fff; text-alig=
 n: left; font-size: 14px; line-height: 1.5;">
       <div style=3D"text-align: center;">
-        <img src=3D"https://coder.com/coder-logo-horizontal.png" alt=3D"Cod=
-er Logo" style=3D"height: 40px;" />
+        <img src=3D"https://coder.com/coder-logo-horizontal.png" alt=3D"Log=

```

Comment:
Can we set alternative text to the app name?

Reply:
I changed :

- Added the app_name as the alt of img.
- Remove the app_name from the email header as discussed with @dannykopping